templates/omero-web-config-for-webapps.j2

```diff
@@ -14,6 +15,8 @@ config append -- omero.web.open_with '["omero_fpbioimage", "fpbioimage_index", {
 # iviewer
 config append -- omero.web.apps '"omero_iviewer"'
 config append -- omero.web.open_with '["omero_iviewer", "omero_iviewer_index", {"supported_objects":["images", "dataset", "well"], "script_url": "omero_iviewer/openwith.js", "label": "OMERO.iviewer"}]'
+# set iviewer as a default viewer
+config set -- omero.web.viewer.view: omero_iviewer.views.index
```

Comment:
Need to remove the `:`, otherwise this will create a property called `omero.web.viewer.view:` which will be ignored.

(Would be nice if `omero config` has some sort of syntax rules so it could detect errors like this)

Reply:
@manics yes, thanks, sorry, removed.